### PR TITLE
Curve2 - Generic

### PR DESCRIPTION
**Pull request by XinYenFon**
Please read commit note(s) for detailed info.

In this PR, I added a todo (or more like discussion point), this seems like an old code from SMF 2.0, considering SMF 2.1 no longer showing post data as tabular data. Question is, should we go ahead a rewrite section or leave it as is.

### CODE REVIEW

**Comment by colinschoen** [comment]
@Gwenwyfar I'll do a sanity check after you.

File: Themes/default/css/index.css
Comment (Gwenwyfar) [inline Themes/default/css/index.css]: Can you move this somewhere more specific? It's going to add odd margins elsewhere (ie: post checkboxes)

**Review by Gwenwyfar — COMMENTED** [review]
Sorry for the wait, busy week. Rest looks good.

**Review by colinschoen — APPROVED** [review]
LGTM approval - pending Gwenwyfar's change.

**Review by Gwenwyfar — APPROVED** [review]

**Comment by XinYenFon** [comment]
I have no idea why HHVM failed 😆 

edit: thank mittens @jdarwood007 solved it

**Comment by albertlast** [comment]
@Sesquipedalian @colinschoen why you not merge this pr?

**Comment by Sesquipedalian** [comment]
@colinschoen is travelling, and I haven't paying attention to this one.

**Pull request merged**
